satpy/tests/reader_tests/test_nwcsaf_nc.py

```diff
@@ -161,3 +150,11 @@ def test_scale_dataset_floating(self):
         self.assertNotIn('add_offset', var.attrs)
         self.assertEqual(var.attrs['valid_range'][0], -2000.)
         self.assertEqual(var.attrs['valid_range'][1], 25000.)
+
+
+def _check_area_def(adef):
+    assert adef.proj_dict['h'] > 100e3
+    assert adef.proj_dict['a'] > 10e3
+    assert adef.proj_dict['units'] == 'm'
+    assert min(adef.area_extent) < -100e3
+    assert max(adef.area_extent) > 100e3
```

Comment:
why not checking the actual values anymore? that would avoid small one-pixel-size displacement error bugs in the future...